content/streamlit-cloud/manage-your-account/sign-in-sign-out.md

```diff
@@ -13,17 +13,17 @@ Once you've created your account, you can sign in to <a href="https://share.stre
 
 1. Visit <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a> and click "**Continue with Google**".
 
-    <div style={{ maxWidth: '80%', margin: 'auto' }}>
-        <Image src="/images/streamlit-community-cloud/sign-in-Google-1.png" alt="Sign in to Streamlit Community Cloud with Google" />
-    </div>
+<div style={{ maxWidth: '80%', margin: 'auto' }}>
+    <Image src="/images/streamlit-community-cloud/sign-in-Google-1.png" alt="Sign in to Streamlit Community Cloud with Google" />
+</div>
```

Comment:
If this `<div>` is a pattern we use often, we should make a component for it.

Reply:
That is also on my want list, yes! 😁